settings/base.py

```diff
@@ -190,3 +190,15 @@
 csv = Csv(cast=lambda s: tuple(s.split(':')))
 
 ADMINS = csv(os.environ.get('ADMINS', ''))
+
+SWAGGER_SETTINGS = {
+    'SECURITY_DEFINITIONS': {
+        'basic': {
+            'type': 'basic'
+        }
```

Comment:
@nelmandela we need token authentication for this

Reply:
on it